assets/js/components/Banner/index.stories.js

```diff
@@ -22,19 +22,16 @@
 import { Fragment } from 'react';
 import Banner from '.';
 import { Cell, Grid, Row } from '../../material-components';
-// TODO: This rule should be updated to allow query parameters in import paths.
-/* eslint-disable import/no-unresolved */
-import AdSenseDesktopSVG from './../../../svg/graphics/banner-adsense-setup-cta.svg?url';
-import AdSenseMobileSVG from './../../../svg/graphics/banner-adsense-setup-cta-mobile.svg?url';
-import AdBlockingRecoveryDesktopSVG from './../../../svg/graphics/banner-ad-blocking-recovery-setup-cta.svg?url';
-import AdBlockingRecoveryMobileSVG from './../../../svg/graphics/banner-ad-blocking-recovery-setup-cta-mobile.svg?url';
-import ConversionsDesktopSVG from './../../../svg/graphics/banner-conversions-setup-cta.svg?url';
-import ConversionsMobileSVG from './../../../svg/graphics/banner-conversions-setup-cta-mobile.svg?url';
-import EnhancedMeasurementDesktopSVG from './../../../svg/graphics/banner-enhanced-measurement-setup-cta.svg?url';
-import EnhancedMeasurementMobileSVG from './../../../svg/graphics/banner-enhanced-measurement-setup-cta-mobile.svg?url';
-import SignInWithGoogleDesktopSVG from './../../../svg/graphics/banner-sign-in-with-google-setup-cta.svg?url';
-import SignInWithGoogleMobileSVG from './../../../svg/graphics/banner-sign-in-with-google-setup-cta-mobile.svg?url';
-/* eslint-enable import/no-unresolved */
+import AdSenseDesktopSVG from '@/svg/graphics/banner-adsense-setup-cta.svg?url';
+import AdSenseMobileSVG from '@/svg/graphics/banner-adsense-setup-cta-mobile.svg?url';
+import AdBlockingRecoveryDesktopSVG from '@/svg/graphics/banner-ad-blocking-recovery-setup-cta.svg?url';
+import AdBlockingRecoveryMobileSVG from '@/svg/graphics/banner-ad-blocking-recovery-setup-cta-mobile.svg?url';
+import ConversionsDesktopSVG from '@/svg/graphics/banner-conversions-setup-cta.svg?url';
+import ConversionsMobileSVG from '@/svg/graphics/banner-conversions-setup-cta-mobile.svg?url';
+import EnhancedMeasurementDesktopSVG from '@/svg/graphics/banner-enhanced-measurement-setup-cta.svg?url';
+import EnhancedMeasurementMobileSVG from '@/svg/graphics/banner-enhanced-measurement-setup-cta-mobile.svg?url';
+import SignInWithGoogleDesktopSVG from '@/svg/graphics/banner-sign-in-with-google-setup-cta.svg?url';
+import SignInWithGoogleMobileSVG from '@/svg/graphics/banner-sign-in-with-google-setup-cta-mobile.svg?url';
```

Comment:
While it requires a few more changes, the imports here are strings, not component types so the name should reflect this to make it explicit while establishing the convention to use going forward.

Reply:
Updated, please let me know if they look better now, thank you!